docs/en_US/reference/experiment_config.rst

```diff
@@ -609,16 +647,16 @@ type: ``str``
 default: ``"msranni/nni:latest"``
 
 
-nniManagerStorageMountPoint
----------------------------
+localStorageMountPoint
+""""""""""""""""""""""
 
-`Mount point <path>`_ of storage service (typically NFS) on current machine.
+:ref:`Mount point <path>` of storage service (typically NFS) on current machine.
```

Comment:
what is the rendering result of ":ref:`Mount point <path>`"?

Reply:
here https://github.com/microsoft/nni/pull/3711/files/5f2214494067334a693f5dd297c191404cf8a782#r644440498
(``..path``)